api/EXT_texture_filter_anisotropic.json

```diff
@@ -6,73 +6,73 @@
         "support": {
           "chrome": [
             {
-              "version_added": true
+              "version_added": "34"
             },
             {
-              "version_added": true,
+              "version_added": "35",
```

Comment:
This seems a bit implausible, that the prefixed variant would be shipped *after* the unprefixed variant. Can you confirm these results? I found https://bugs.chromium.org/p/chromium/issues/detail?id=118245 but would have to dig deeper to be sure about the versions from source code alone.

Reply:
I thought it was weird as well, but alas, based upon manual testing, the prefixed variant did come a version later.

Reply:
This web platform thing is weird...